### ISSUE TITLE
automaticly set AllowSpectatorJoin to false

### ISSUE THREAD

**Issue by abma** [springlobby/springlobby#560]
https://springrts.com/phpbb/viewtopic.php?f=12&t=33869

**Comment by specing** [comment]
Just setting this would make all players and spectators that were present when hosted battle started connect, while everyone arriving afterwards would be disallowed joining, correct?

**Comment by abma** [comment]
yep: it requires to add new players via autohost interface when a game is already running.
